### PR TITLE
[WIP] Pipelines Gherkin features - Design [ODC-3856]

### PR DESCRIPTION
**Pull request by makambalaji**
Included features related to pipelines

### CODE REVIEW

**Comment by makambalaji** [comment]
**Note**: Knative scenarios will be pushed to the knative folder in future, For now I want to keep them at one place.

File: frontend/packages/dev-console/integration-tests/features/pipelines-secrets.feature [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Background:
+    Given user logged into the openshift application
+    And openshift cluster is installed with pipeline operator
+    And user is at the project namespace "AUT_MB_Demo" in dev perspecitve
+
+
+@regression, @smoke
+Scenario: Add Secret details : P-011-TC01
+   Given user able to see pipeline with git resources in piepiles page
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/pipelines-secrets.feature:12]: *pipelines

File: frontend/packages/dev-console/integration-tests/features/login.feature [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Feature: Login to Application
+
+    As a valid user
+    I want to log in to the Application
+
+    @smoke, @odc-1255
+    Scenario Outline: Valid Login for "<userName>" : P-02-TC01
+        Given I open the login Page
+        When I enter a username "<userName>"
```
Comment (gajanan194) [inline frontend/packages/dev-console/integration-tests/features/login.feature:9]: Can you please update it with the "User" and not "I"

File: frontend/packages/dev-console/integration-tests/features/operators.feature [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+   When user installs the pipeline operator with default values
+   Then page redirects to Installed operators
+   And page will contain OpenShift Pipeline Operator 
+
+
+@regression, @smoke
+Scenario: Uninstall the Pipeline Operator from Operator Hub page : P-013-TC01, P-013-TC02
+   Given user is at OpenShift Pipeline Operator subscription page
+   When user uninstalls the pipeline operator from right side pane
+   And clicks on Unistall button present in popup with header message "Uninstall Operator?"
```
Comment (gajanan194) [inline frontend/packages/dev-console/integration-tests/features/operators.feature:29]: The "Then" statement should have the 'should'. Also, these are sentences and it should start with capital letter. 
E.g. Line 21: User should redirect to Installed Operators page.

Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/operators.feature:29]: As part of best practices, Better not to use should [Assertion statements] and don't use capital letters after Given, When Then statement 
"page redirects" is fine, the reason is page is redirecting, it's an outcome from above actions [when statement]

File: frontend/packages/dev-console/integration-tests/features/login.feature (hunk 2 above)
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/login.feature:9]: I will update

File: frontend/packages/dev-console/integration-tests/features/pipelines-secrets.feature (hunk 1 above)
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/pipelines-secrets.feature:12]: spell mistake in line no.6 - updated

**Pull request closed without merging**